Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+def DOCKER_IMG = "registry.fedoraproject.org/fedora:28"
+def DOCKER_ARGS = "--net=host -v /srv:/srv --privileged"
+
+// this var conveniently refers to a location on the server as well as the local dir we sync to/from
+def repo = "${env.ARTIFACT_SERVER_DIR}/repo"
```

Comment:
quick question, where does this env variable get defined? Is it on the jenkins side? =)

Reply:
That's correct, this is defined on the Jenkins side since it depends on where it runs. We'll need to make this more flexible if we want to support e.g. GCS buckets.